### PR TITLE
chore: fix some comments

### PR DESCRIPTION
**Pull request by riskrose**
fix some comments

### CODE REVIEW

**Review by wmontwe — APPROVED** [review]
Thank you.

File: ui-utils/LinearLayoutManager/src/main/java/app/k9mail/ui/utils/linearlayoutmanager/LinearLayoutManager.java [hunk 1]
```diff
@@ -1874,7 +1874,7 @@ View findFirstVisibleChildClosestToEnd(boolean completelyVisible,
      *                      (reverseLayout ^ stackFromEnd).
      * @param traverseChildrenInReverseOrder True if the children should be traversed in reverse
      *                                       order (stackFromEnd).
-     * @return A View that can be used an an anchor View.
+     * @return A View that can be used an anchor View.
```
Comment (cketti) [inline ui-utils/LinearLayoutManager/src/main/java/app/k9mail/ui/utils/linearlayoutmanager/LinearLayoutManager.java:1877]: This should be "…used as an anchor View."

Comment (kewisch) [inline ui-utils/LinearLayoutManager/src/main/java/app/k9mail/ui/utils/linearlayoutmanager/LinearLayoutManager.java:1877]:
```suggestion
     * @return A View that can be used as an anchor View.
```

Comment (kewisch) [reply ui-utils/LinearLayoutManager/src/main/java/app/k9mail/ui/utils/linearlayoutmanager/LinearLayoutManager.java:1877]: Adding this as a suggestion so it is easier to merge

**Pull request merged**